run/testing/helloworld.e2e_test.go

```diff
@@ -36,30 +34,17 @@ func TestHelloworldService(t *testing.T) {
 	}
 	defer service.Clean()
 
-	req, err := service.NewRequest("GET", "/")
+	resp, err := service.Request("GET", "/")
+	out, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		t.Fatalf("service.NewRequest: %v", err)
+		t.Errorf("ioutil.ReadAll: %v", err)
```

Comment:
nit: replace use of `ioutil` with `io`.

Reply:
is this safe to do while we still (nominally) support versions < 1.18, or should we wait?

Reply:
We discussed this OOB, and decided to wait on updating the package here until 1.18 is our minimum supported version.